pkg/clioptions/suiteselection/feature_filter.go

```diff
@@ -68,16 +67,11 @@ func (f *featureGateFilter) includeTest(name string) bool {
 		featureGates = append(featureGates, featureGate)
 	}
 
-	hasAnyDisabledFeatureGates := f.disabled.HasAny(featureGates...)
-	hasAllEnabledFeatureGates := f.enabled.HasAll(featureGates...)
-	shouldIncludeTest := !hasAnyDisabledFeatureGates && hasAllEnabledFeatureGates
-	if shouldIncludeTest {
-		logrus.Infof("Including test: %q", name)
-	} else {
-		logrus.Infof("Excluding test: %q (requires feature-gates: %v)", name, featureGates)
+	if f.disabled.HasAny(featureGates...) {
+		return false
 	}
 
-	return shouldIncludeTest
+	return f.enabled.HasAll(featureGates...)
```

Comment:
Reverted to pre-patch state since this [isn't needed anymore](https://redhat-internal.slack.com/archives/CE4L0F143/p1718119344698569?thread_ts=1718116330.342849&cid=CE4L0F143) (`/verify` seems to pass now).